src/main/java/com/example/javaOjt/services/AuthorService.java

```diff
@@ -79,16 +79,29 @@ public GetAuthorsResponse getAuthorsList(@Nullable AuthorSortBy attribute, @Null
    * Saves a new author to the DB with the given String as the name.
    *
    * @param name the name of the author to be added
-   * @return the Author that is saved to the DB
+   * @return GetAuthorResponse of the author that was saved to the DB
    */
   public GetAuthorResponse postAuthor(String name) {
     Author newAuthor = new Author();
     newAuthor.setName(name);
     return new GetAuthorResponse(authorRepository.save(newAuthor));
   }
 
+  /**
+   * Update the name of the author of the given id.
+   *
+   * @param id   the id of the author that is to be updated
+   * @param name new name of the author
+   * @return GetAuthorResponse of the author that was updated
+   */
   public GetAuthorResponse patchAuthor(Integer id, String name) {
```

Comment:
`@Transaction` をサービス層のメソッドにつけましょう！

```suggestion
  @Transactional
  public GetAuthorResponse patchAuthor(Integer id, String name) {
```